posts/month-in-mathlib-may-2024.md

```diff
@@ -0,0 +1,194 @@
+  * Mathlib finally knows about Hamiltonian paths and cycles thanks to a team effort started at Lean for the Curious Mathematician 2023 in Düsseldorf by Rishi Mehta and Linus Sommer under the supervision of Bhavik Mehta, and recently continued by Lode Vermeulen in [PR #7102](https://github.com/leanprover-community/mathlib4/pull/7102).
+
+* Condensed mathematics.
+  * The work towards getting the basics of condensed mathematics into Mathlib started about a year ago at a workshop in Copenhagen organized by Dagur Asgeirsson. The participants made great progress and proved results like the characterization of condensed sets as finite-product-preserving presheaves on `Stonean`. Since then, Dagur has been moving the material to Mathlib and developing it further. This month landed
+    * [PR #11586](https://github.com/leanprover-community/mathlib4/pull/11586) defining light condensed objects. This is a concept introduced by Clausen and Scholze less than a year ago, and is an improvement over condensed sets in that it removes certain size-issues, as it is a sheaf over an essentially small site.
+    * [PR #12870](https://github.com/leanprover-community/mathlib4/pull/12870) proving the explicit sheaf condition for condensed objects in a very general target category. Previously, we only had this in certain concrete categories.
+  * Dagur Asgeirsson
+    * [PR #9513](https://github.com/leanprover-community/mathlib4/pull/9513) :: feat(LightProfinite): (co)limits
+    * [PR #11585](https://github.com/leanprover-community/mathlib4/pull/11585) :: feat(LightProfinite): `LightProfinite` is precoherent
+
+* Measure Theory.
+  * Rémy Degenne
+    * [PR #12759](https://github.com/leanprover-community/mathlib4/pull/12759) :: feat: every s-finite measure is absolutely continuous w.r.t. some finite measure
+    * [PR #12467](https://github.com/leanprover-community/mathlib4/pull/12467) :: feat: add algebra of sets
+  * Kalle Kytölä
+    * [PR #11815](https://github.com/leanprover-community/mathlib4/pull/11815) :: feat: Embed a space into probability measures on it.
+
+* Number Theory.
+  * David Loeffler
+    * [PR #12265](https://github.com/leanprover-community/mathlib4/pull/12265) :: feat(NumberTheory/LSeries): Even Hurwitz zeta functions (II)
+    * [PR #12779](https://github.com/leanprover-community/mathlib4/pull/12779) :: feat(NumberTheory/LSeries): Odd Hurwitz zeta functions
+    * [PR #12897](https://github.com/leanprover-community/mathlib4/pull/12897) :: feat (NumberTheory/LSeries): Hurwitz zeta
+    * [PR #13347](https://github.com/leanprover-community/mathlib4/pull/13347) :: feat(NumberTheory/LSeries): special values of Hurwitz zeta
+    * [PR #13273](https://github.com/leanprover-community/mathlib4/pull/13273) :: feat(NumberTheory/LSeries): Riemann zeta as special case of Hurwitz
+    * [PR #12923](https://github.com/leanprover-community/mathlib4/pull/12923) :: feat (NumberTheory/Harmonic): compute Gamma'(1/2)
+  * Riccardo Brasca
```

Comment:
This can be removed, it is not interesting (of course the final proof will be!).